### ISSUE TITLE
club together with greatdevelopers.github.io 

### ISSUE THREAD

**Issue by amarjeetkapoor1** [GreatDevelopers/gd_workshops#1]
I think all the .pdf file here  should be converted into .md files and we should add them to http://greatdevelopers.github.io/

**Comment by mandeeps708** [comment]
Plus some other ones we find on GD archive (Google Docs) like one was re-shared by kamalpreet i.e. 
https://docs.google.com/document/d/163P8rtNQpd_OUttfa--mjLDRFbquiaw47cPWtQK3LXY/edit?usp=sharing

**Comment by amarjeetkapoor1** [comment]
Yes, Just share here all links that you get. Anyone with task related to markdown could convert them.
